static/templates/base.html

```diff
@@ -118,7 +118,7 @@
 
     <li id="section-about" class="navigation-primary__item"><a href="http://iatistandard.org/en/about/">About</a></li>
     <li id="section-standard" class="navigation-primary__item"><a href="http://iatistandard.org/en/iati-standard">IATI Standard</a></li>
-    <li id="section-data" class="navigation-primary__item"><a href="http://iatistandard.org/en/using-data">Using IATI Data</a></li>
+    <li id="section-data" class="navigation-primary__item"><a href="http://iatistandard.org/en/using-data">Using Data</a></li>
```

Comment:
Not sure about this change. The linked page has the headline “Using IATI Data”, and refers to “IATI data” (rather than “data”) throughout.

Reply:
Oh I see – it’s for consistency with the menu on the IATI standard site. In that case it’s fine I guess.